compiler/src/import_map.rs

```diff
@@ -1,8 +1,6 @@
 use indexmap::IndexMap;
```

Comment:
Instead of re-implementing import maps, use https://crates.io/crates/import_map. This crate is maintained by us: https://github.com/denoland/import_map